src/radical/pilot/agent/launch_method/base.py

```diff
@@ -198,11 +198,16 @@ def get_task_named_env(self, env_name):
         src  = '%s.env'                 %  base
         tgt  = '%s.%s.sh'               % (base, self.name.lower())
 
+        blacklist = self.get_env_blacklist()
+
+        self._log.debug('=== blacklist: %s', blacklist)
```

Comment:
maybe to have it for a higher debug level (not a regular one)?

Reply:
Done